std_msgs/Header header
float32[] controller1_amps
float32[] controller2_amps
